pkg/types/types.go

```diff
@@ -234,39 +262,69 @@ func (s byStepTime) Less(i, j int) bool {
 	return s[i].StepTime < s[j].StepTime
 }
 
-func mergeMetrics(metrics []Metric) Metric {
+func getPointInconsistentValueCount(selectedValue float64, values []float64) int {
+	valuesCount := len(values)
+	if valuesCount == 0 {
+		// No value for the point
+		return 0
+	}
+
+	valuesToCount := make(map[float64]int)
+	for _, val := range values {
+		valuesToCount[val]++
+	}
+
+	return valuesCount - valuesToCount[selectedValue]
```

Comment:
It seems return value is not important. Perhaps we can return boolean instead?

Reply:
Yeah. it's all because at first, the function was doing a bit more. right now, it totally makes sense.